flink-dist/pom.xml

```diff
@@ -399,23 +399,6 @@ under the License.
 					<name>include-hadoop</name>
 				</property>
 			</activation>
-			<dependencies>
-				<!--
-                    The Hadoop 2 Uber jar should not go into the Flink dist jar, but
-                    sit next to it. Hence, we set it to 'provided' here.
-                -->
-
-				<dependency>
-					<groupId>org.apache.flink</groupId>
-					<artifactId>flink-shaded-hadoop2-uber</artifactId>
```

Comment:
we want to retain the ability to bundle hadoop, for the time being at least, as such this should stay (without it the entire profile makes no sense anyway)